playbooks/deploy-rock.yml

```diff
@@ -1136,6 +1136,17 @@
       group: root
     when: with_kibana
 
+  - name: Set initial Kibana credentials
+    shell: >
+      kibuser=$(getent passwd 1000 | awk -F: '{print $1}')
```

Comment:
Does the Initial Setup script force a user 1000 creation? If not, maybe this (need to validate; not sure if error propagates)
```
kibuser=$((getent passwd 1000 | awk -F: '{print $1}') || echo 'root')
```